include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -1344,7 +1344,16 @@ elements along the K dim, or they use all elements of the tensor along the K dim
   let genVerifyDecl = 1;
   let extraClassDeclaration = extraDistributedDeclaration # [{
     SmallVector<unsigned> getContigPerThread() {
-      return getSizePerThread();
+      auto rank = getWarpsPerCTA().size();
+      assert(rank == 2 || rank == 3);
+      SmallVector<unsigned> contigPerThread(rank, 1);
+      auto kWidth = getKWidth();
+      assert(kWidth != 0 && "Do not support kWidth=0");
```

Comment:
This should be in the verifier.

Reply:
ah, I see that for mmav1 we set kWidth = 0

Reply:
Yeah, I think our plan to take out mmav1 support in the end. Just to support FMA dot for old architectures